esp/services/ws_fs/ws_fsService.cpp

```diff
@@ -1664,8 +1672,12 @@ bool CFileSprayEx::onDeleteDFUWorkunits(IEspContext &context, IEspDeleteDFUWorku
         StringArray & wuids = req.getWuids();
         for(unsigned i = 0; i < wuids.ordinality(); ++i)
         {
-            if (markWUFailed(factory, wuids.item(i)))
-                factory->deleteWorkUnit(wuids.item(i));
+            const char* wuid = wuids.item(i);
+            if (markWUFailed(factory, wuid))
+            {
+                factory->deleteWorkUnit(wuid);
+                PROGLOG("%s deleted", wuid);
```

Comment:
Might want to be a bit more descriptive on these deletes... so we can easily find the line that did the delete..

 PROGLOG("DeleteDFUWorkunits: %s deleted", wuid);?